documents/installation-list.md

```diff
@@ -0,0 +1,47 @@
+# Installation List
+
+## For OSX Users:
+
+- Install [Brew](http://brew.sh/)
+- Optional: iTerm2
+
+## For Linux Users:  
+
+| Distro        | Base           | Package Manager | 
+|-------------|-------------|-----  
+| [Ubuntu](https://www.ubuntu.com/) | [Debian](https://www.debian.org/) | [APT](https://help.ubuntu.com/lts/serverguide/apt.html) | 
+| [Linux Mint](https://linuxmint.com/) | [Debian](https://www.debian.org/) | [APT](https://help.ubuntu.com/lts/serverguide/apt.html) |  
+
+- Ubuntu and Linux Mint are probably the two most used desktop Linux distros out there. Mint is becoming increasingly popular and in a lot of ways, is more beginner-friendly than Ubuntu. It also uses up less memory resource (the most lightweight Mint edition is XFCE), and will be better suited to slower machines.
+
+## For Windows Users:
+
+- For the course you'll **need** to install a Linux operating system on your machine and dual boot - students in the past who have tried to use Windows always encounter big setbacks with installation problems and general incompatibility issues.
+- It's not as scary as it sounds! There are a lot of articles online to help you. Due to a lot of different OS configurations you may be best off searching google for your specific OS + distro (eg: "dual boot windows 10 and Ubuntu"). However here are some links that may be useful:
+
+- [How to install Ubuntu 18.04 alongside Windows 10](https://askubuntu.com/questions/1031993/how-to-install-ubuntu-18-04-alongside-windows-10) - useful at time of posting (mid 2018)
+- [Dual boot Linux](https://www.howtogeek.com/214571/how-to-dual-boot-linux-on-your-pc/) 
+- We shall have a couple of FAC students and alumni available at the installation party to guide you through installing linux. Please come along if you have Windows OS.
+- This [Lifewire article](https://www.lifewire.com/beginners-guide-to-linux-4090233) is informative and accessible, and [this PCWorld article](https://www.pcworld.com/article/2918397/how-to-get-started-with-linux-a-beginners-guide.html) has more info on installing Linux.
+- Dual boot [Windows 10 + Linux Mint](https://itsfoss.com/guide-install-linux-mint-16-dual-boot-windows/), and [Windows 10 + Ubuntu](https://itsfoss.com/install-ubuntu-1404-dual-boot-mode-windows-8-81-uefi/)
+- Some extra info on [APT](https://itsfoss.com/apt-command-guide/), the command-line tool for the Debian packaging system.
+- Finally please do not hesistate to come to a meetup or contact us on slack if you are having trouble, we can help!
```

Comment:
This feels like a lot of resources to throw at them. Is there a single definitive guide we can provide?